docs/advanced-usage/nesting-aggregate-metrics.md

```diff
@@ -0,0 +1,118 @@
+---
+icon: kolena/nest-16
```

Comment:
really really nit and optional:

None builtin icons I quickly checked totally make sense or intuitive, but any of `Array`, `tree diagram`, or `layout: hierarchy` feels slightly better?

Reply:
Updated to `diagram-tree` 👍